### PR TITLE
Add configs for BOME, MEME, FLOKI, MEW

### PR DESCRIPTION
**Pull request by dmax10**

### CODE REVIEW

**Review by SentinelDev1 — APPROVED** [review]

File: config/markets.ts [hunk 1]
```diff
@@ -1718,6 +1718,138 @@ const config: {
...
+      ...borrowingRateConfig_HighMax_WithHigherBase,
+
+      negativePositionImpactFactor: exponentToFloat("5e-10"),
+      positivePositionImpactFactor: exponentToFloat("2.5e-10"),
+      positionImpactExponentFactor: exponentToFloat("2.2e0"),
+
+      negativeSwapImpactFactor: exponentToFloat("5e-9"),
+      positiveSwapImpactFactor: exponentToFloat("2.5e-9"),
+
+      minCollateralFactor: percentageToFloat("0.02%"), // 50x leverage
```
Comment (gmxer) [inline config/markets.ts]: should be 2%, otherwise it's 1 / 0.0002 = 5000x leverage

Comment (gmxer) [reply config/markets.ts]: and same for MEME

**Review by gmxer — CHANGES_REQUESTED** [review]

File: config/markets.ts (hunk 1 above)
Comment (dmax10) [reply config/markets.ts]: updated

**Review by gmxer — APPROVED** [review]
there's a pending question regarding minCollateralFactor for BOME/MEME but it's still good to be listed that way, we will clarify that later

**Pull request merged**